figures/templates/figures/admin_dropdown_filter.html

```diff
@@ -0,0 +1,28 @@
+{# Attribution: Original code here: https://github.com/mrts/django-admin-list-filter-dropdown #}
+
+{% load i18n %}
+<script type="text/javascript">
+    var go_from_select = function(opt) { window.location = window.location.pathname + opt };
```

Comment:
```suggestion
    var figures_go_from_select = function(opt) { window.location = window.location.pathname + opt };
```

Just a precautionary prefix.

Reply:
Good call, @OmarIthawi !